### PR TITLE
Adding Discord Checkpoint

### PR DESCRIPTION
**Pull request by Crypti-x**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

**Comment by lepon01** [comment]
This bot may take longer to look at...

**Comment by lepon01** [comment]
That's actually pretty cool.

File: data/en/bots/null.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Secure your Discord Server - the strongest ban enforcement system, protection
+  against scraping and bots - without privacy issues.
+github: null
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=496677692284993546&scope=bot&permissions=355571&redirect_uri=https%3A%2F%2Fcheckpoint.discord.id%2Fadded%2F&response_type=code
+nsfw: false
+pagename: Discord Checkpoint
+prefix: cp!
+support: 'https://discord.gg/bGcjZRz'
```
Comment (lepon01) [inline data/en/bots/null.md]: You may wish to create another server just for support - It just opens the checkpoint with no channels to talk in

File: data/en/bots/null.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (lepon01) [inline data/en/bots/null.md]: The name of the file needs to match the ID of your bot.

**Comment by Andre601** [comment]
So if I understand that correctly is that a sort of redirect thing with some checks?

**Comment by lepon01** [comment]
yea

File: data/en/bots/null.md (hunk 1 above)
Comment (Crypti-x) [reply data/en/bots/null.md]: Hm, is this a actual problem?

File: data/en/bots/null.md (hunk 2 above)
Comment (Crypti-x) [reply data/en/bots/null.md]: I added it via the website, so not my fault

Comment (lepon01) [reply data/en/bots/null.md]: Did you fill in the `Bot User ID` field?
It should have worked

**Comment by Crypti-x** [comment]
Ok, fixed

File: data/en/bots/null.md (hunk 1 above)
Comment (lepon01) [reply data/en/bots/null.md]: Well the button is for technical support with the bot

If users can't find technical support, it's not useful as a button

**Comment by lepon01** [comment]
image
This probably shouldn't happen if I leave before the timer runs out

**Pull request merged**